dotcom-rendering/src/web/components/FixedSmallSlowIII.tsx

```diff
@@ -15,11 +15,11 @@ export const FixedSmallSlowIII = ({
 	containerPalette,
 	showAge,
 }: Props) => {
-	const smallTrails = trails.slice(0, 3);
+	const primary = trails.slice(0, 3);
 
 	return (
 		<UL direction="row">
-			{smallTrails.map((trail, index) => {
+			{primary.map((trail, index) => {
```

Comment:
We might as well inline that directly?

```suggestion
			{trails.slice(0, 3).map((trail, index) => {
```

Reply:
I'm somewhat of a fan of keeping slices explicit at the start of a container, I feel like it makes it clearer how a container is using cards!